spec/pdc/v1/multi_destination_spec.rb

```diff
@@ -23,5 +23,9 @@
       count = multi_destinations.where(active: false).count
       count.must_equal 0
     end
+
+    it 'return empty data for specific global component and origin repo release' do
+      multi_destinations.where(active: true, global_component: 'ceph', origin_repo_release_id: 'ceph-1.3', page_size: -1).all!.must_equal []
```

Comment:
If we use "all!" here, the page_size has been set by -1 by default.

Reply:
Ack. Fixed